packages/python-packages/api-stub-generator/apistub/_stub_generator.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import glob
+import sys
+import os
+import argparse
+import astroid
```

Comment:
Caught this offhand, I'm not certain how many more need to be cleaned up. Use a `pylint` pass for `unused import`